mappings/net/minecraft/entity/ai/FuzzyPositions.mapping

```diff
@@ -41,8 +41,8 @@ CLASS net/minecraft/class_5535 net/minecraft/entity/ai/FuzzyPositions
 		ARG 1 horizontalRange
 		ARG 2 verticalRange
 		ARG 3 startHeight
-		ARG 4 xDirection
-		ARG 6 zDirection
+		ARG 4 directionX
+		ARG 6 directionZ
```

Comment:
Sorry, my bad names! should have named them the way you do especially given the doc says they should be interpreted together as one "direction"